internal/fabric/testing_helpers/env_data.go

```diff
@@ -15,6 +16,10 @@ const (
 	FabricConnectionsTestDataEnvVar = "TF_ACC_FABRIC_CONNECTIONS_TEST_DATA"
 )
 
+const (
+	FabricSubscriptionEnvVar = "TF_ACC_FABRIC_MARKET_PLACE_SUBSCRIPTION_ID"
```

Comment:
You can add this to the const statement above. You don't need to separate them. It can house many different constants.